app/swagger/schemas/service_history.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Swagger
+  module Schemas
+    class ServiceHistory
```

Comment:
Do we see any other instances where we're going to use a `ServiceHistory` schema in another response yet? Based on our conversations yesterday I thought we were going to avoid using a model until we decided to repeat it. 

Not a deal breaker, for sure, but I also wonder if it makes sense for the `ServiceHistory` model to have the ActiveModel serializer wrapper properties:

`ServiceHistory: [{ branch_of_service: string, begin_date: string, end_date: string }]`